esp-alloc/src/macros.rs

```diff
@@ -41,3 +41,16 @@ macro_rules! psram_allocator {
         }
     }};
 }
+
+/// Gets the usage stats for the current memory heap
```

Comment:
Let's remove this macro, in favour of documenting `esp_alloc::HEAP.stats()` in the lib level docs